schema.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(225),
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories(
+    id INT  PRIMARY KEY,
+    admited_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(225)
+);
+
+CREATE TABLE envoices (
+    id INT PRIMARY KEY,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE traitments(
+    id INT PRIMARY KEY,
+    type VARCHAR(225),
+    name VARCHAR(225)
+);
+
+CREATE TABLE  invoice_items (
+   id INT PRIMARY KEY,
+   unit_price DECIMAL,
+   quantity INT,
+   total_price DECIMAL,
+   invoice_id INT REFERENCES envoices(id),
+   traitment_id  INT REFERENCES traitments(id)
```

Comment:
- [ ] Based on the requirements, you ought to create indexes on all foreign keys created. Kindly implement this feature for all foreign keys in this schema. 👍 😎